[EDI] UDES-14 - Gateway menu is slow when lots of transfers

vvasoyav opened the pull request:
EDI gateway menu is slow down when lots of edi transfers happens. so it is resolved by improve the manage transfer.

Story :: SE-18

Signed off By :: vishal.vasoya@unipart.com

rajeeshknambiar commented on addons/edi/models/edi_gateway.py:
```diff
@@ -193,8 +193,17 @@ def _compute_path_count(self):
+        transfers_data = self.env['edi.transfer'].read_group(
+        [('gateway_id', 'in', self.ids)],
+        ['gateway_id'], ['gateway_id'])
+        transfer_count_dict = {data['gateway_id'][0]: data['gateway_id_count'] for data in transfers_data}
```
Nit: black formatting

```py
        transfers_data = self.env["edi.transfer"].read_group(
            [("gateway_id", "in", self.ids)], ["gateway_id"], ["gateway_id"]
        )
        transfer_count_dict = {
            data["gateway_id"][0]: data["gateway_id_count"] for data in transfers_data
        }
```

vvasoyav replied:
Yeah Changed.

armandcela replied:
I see you pushed a new commit, but don't see that pushed suggested changes. Still black formatting is not applied and the other suggestions from 11 MR-s which they are same for this MR as well. Am i missing smth?

vvasoyav replied:
I have applied chanes in botn MRs can u please check it again and in 14 only one commit is there but for 11 there 2 commits.

vvasoyav committed:
[EDI] UDES-14 - Gateway menu is slow when lots of transfers

EDI gateway menu is slow down when lots of edi transfers happens.
so it is resolved by improve the manage transfer.

Story :: SE-18

Signed-off-by: vishal vasoya <vishal.vasoya@unipart.com>
